README.md

```diff
@@ -49,6 +49,17 @@ curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh
 
 Alternatively, **any supported platform**: Grab the [latest release](https://github.com/Kha/elan/releases/latest) for your platform, unpack it, run it in a terminal, and follow the printed instructions.
 
+# Prerequisites
+
+On some systems `elan` will not work out of the box:
+
+* Windows 10
+   * You'll need a unix-like terminal.
+      * "git bash", available via [Git for Windows](https://gitforwindows.org/) works well, or
+      * the terminal from [MSYS2](https://www.msys2.org/), and then run `pacman -S unzip`
+   * [Visual C++ redistributable](https://www.microsoft.com/en-au/download/details.aspx?id=48145) (if you've already installed other software you likely have this).
```

Comment:
@semorrison This should be fixed in elan 0.7.1. Do you still have any computer without the runtime left to give it a test? :)

Reply:
I uninstalled the runtime, and tried again with 0.7.1. It seems to work.

I'm planning on making a fresh windows box sometime anyway, so I'll double check then.